README.md

```diff
@@ -1116,3 +1118,210 @@ mutation deletePerson {
   }
 }
 ```
+
+# Encounter
+## Query Encounter (Single)
```

Comment:
@Kolbein  please move all the examples to Wiki page.  https://github.com/sureshHARDIYA/intromat-fhir/wiki. 
- Create a page called `Observation` and make sections (Query/Creation/Updating/Deletion)
- Remove these examples from readme.MD file.

Reply:
@sureshHARDIYA I've created a page on the wiki called "Resources" containing everything from the README.md file, including what @Kolbein has pushed to his branch. Is that OK or would you rather have separate pages for each resource?

Reply:
@martinheitmann  please create a separate page for each resource. Having everything in one would be messy and moreover, we would want to include as much examples as possible in each resources for reference. This can later be used in your report as well. So, the wiki will act as a source of proof for your work in any case.

Reply:
@sureshHARDIYA Got it. I'll move the existing examples into separate pages then.

Reply:
@sureshHARDIYA All the examples has been moved into separate wiki pages, removing them from the README.md still remains though.

Reply:
Remove from README.md file, please.